src/main/java/com/feywild/feywild/tag/ModBlockTags.java

```diff
@@ -9,5 +9,8 @@
 public class ModBlockTags {
 
     public static final Tag.Named<Block> FEY_LOGS = BlockTags.bind(new ResourceLocation(FeywildMod.getInstance().modid, "fey_logs").toString());
-
+    public static final Tag.Named<Block> AUTUMN_LOGS = BlockTags.bind(new ResourceLocation(FeywildMod.getInstance().modid, "autumn_logs").toString());
+    public static final Tag.Named<Block> SPRING_LOGS = BlockTags.bind(new ResourceLocation(FeywildMod.getInstance().modid, "spring_logs").toString());
+    public static final Tag.Named<Block> SUMMER_LOGS = BlockTags.bind(new ResourceLocation(FeywildMod.getInstance().modid, "summer_logs").toString());
+    public static final Tag.Named<Block> WINTER_LOGS = BlockTags.bind(new ResourceLocation(FeywildMod.getInstance().modid, "winter_logs").toString());
```

Comment:
Since 1.17 you can use `FeywildMod.getInstance.resource("xxxx_logs")` to create the `ResourceLocation which is a bit more elegant.